.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_USER
+# DOCKER_PASS
+#
+
+version: 2
+jobs:
+  build:
+    machine:
+      enable: true
+    working_directory: ~/mozilla/telemetry-analysis-service
+    steps:
+      - checkout
+      - run: ./bin/build
+
+  test:
+    machine:
+      enable: true
+    working_directory: ~/mozilla/telemetry-analysis-service
+    steps:
+      - checkout
+      - run: sudo apt-get update
+      - run: sudo apt-get install python-dev
+      - run: sudo pip install --upgrade pip
+      - run: pip install docker-compose
+      - run: docker info
+      - run: docker --version
+      - run: ./bin/test
+
+  deploy:
+    machine:
+      enable: true
+    working_directory: ~/mozilla/telemetry-analysis-service
+    steps:
+      - checkout
+      - run: ./bin/build && ./bin/deploy
+
+workflows:
+  version: 2
+  build-test-deploy:
+    jobs:
+      - build:
+          filters:
+            tags:
+              only: /.*/
+
+      - test:
+          filters:
+            tags:
+              only: /.*/
```

Comment:
I think in the old version we ran build before the test as well, to make sure we didn't break the Docker setup in PRs. Does that do that here as well?

Reply:
in this case, test and build will run as separate circleci builds.  If either fail, then we wont be able to merge the PR

Reply:
Cool!